lib/dist/datadog.yaml

```diff
@@ -104,9 +104,9 @@ api_key:
 # Metadata providers, add or remove from the list to enable or disable collection.
 # Intervals are expressed in seconds. You can also set a provider's interval to 0
 # to disable it.
-metadata_collectors:
-  - name: 'resources'
```

Comment:
What's that for?

Reply:
Apparently it sends some old metadata payload used in the Agent v5
https://github.com/DataDog/datadog-agent/blob/main/pkg/metadata/resources.go#L18-L19

But I only removed it when metadata collection wasn't correctly disabled. I'll put it back.